jsdoc/Home.md

```diff
@@ -169,14 +169,20 @@ The `style-src` directive is required so that the styles for the Google Pay butt
 
 ### 3D Secure Specific Directives
 
-If using [3D Secure](module-braintree-web-drop-in.html#~threeDSecureOptions), include these additional directives:
+If using the [3D Secure component](module-braintree-web_three-d-secure.html) with a 3D Secure 1.0 integration, include these additional directives:
 
 |             | Sandbox                           | Production                    |
 |-------------|-----------------------------------|-------------------------------|
 | script-src  | songbirdstag.cardinalcommerce.com | songbird.cardinalcommerce.com |
-| frame-src   | &#42;.cardinalcommerce.com        | &#42;.cardinalcommerce.com    |
+| frame-src   | &#42;.cardinalcommerce.com        | frame-src *                   |
```

Comment:
random q: does this mean sandbox's CSP is more restrictive than prod?

Reply:
Technically yes, because in sandbox, everything goes through `*.cardinalcommerce.com`'s sandbox. Should probably just set it to be the same for both, just for simplicity. Should also use the html encoded version of `*`

```suggestion
| frame-src   | &#42;                             | &#42;                         |
```

Reply:
Good catch, the same directive should be specified for sandbox. I made that update.